### PR TITLE
Create numpy-accessibility.md

### LINKED ISSUE #4: Article by Mars Lee "Making Numpy accessible: guidelines and tools"
Consider featuring the article by Mars Lee "Making Numpy accessible: guidelines and tools"(https://labs.quansight.org/blog/2021/07/numpy-accessibility-guidelines/).

### PR DESCRIPTION
**Pull request by InessaPawson**
Closes https://github.com/numpy/numpy-newsletter/issues/4.

### CODE REVIEW

**Review by MarsBarLee — COMMENTED** [review]
A great summary of the post! Thank you for highlighting this effort.

File: numpy-feature/numpy-accessibility.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+title: "Making Numpy accessible"
+---
+
+Numpy is now foundational to Python scientific computing. Our efforts reach millions of developers each month. 
+As our user base grows, so does the number of users with disabilities. In the open-source ecosystem, accessibility 
+is often considered low-priority, scary, or someone else’s job. Read the blog post by Mars Lee, a front end developer and designer at Quansight,
```
Comment (jakevandervaate) [inline numpy-feature/numpy-accessibility.md]:
```suggestion
is often considered low-priority, scary, or someone else’s job. Read the blog post by Mars Lee, a front-end developer and designer at Quansight, 
```

File: numpy-feature/numpy-accessibility.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+title: "Making Numpy accessible"
+---
+
+Numpy is now foundational to Python scientific computing. Our efforts reach millions of developers each month. 
+As our user base grows, so does the number of users with disabilities. In the open-source ecosystem, accessibility 
+is often considered low-priority, scary, or someone else’s job. Read the blog post by Mars Lee, a front end developer and designer at Quansight, 
+how she along with Melissa Weber Mendonça championed the efforts to support NumPy users with visual impairments
```
Comment (jakevandervaate) [inline numpy-feature/numpy-accessibility.md]:
```suggestion
about how she and Melissa Weber Mendonça championed the efforts to support NumPy users with visual impairments 
```

File: numpy-feature/numpy-accessibility.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+---
+title: "Making Numpy accessible"
+---
+
+Numpy is now foundational to Python scientific computing. Our efforts reach millions of developers each month. 
+As our user base grows, so does the number of users with disabilities. In the open-source ecosystem, accessibility 
+is often considered low-priority, scary, or someone else’s job. Read the blog post by Mars Lee, a front end developer and designer at Quansight, 
+how she along with Melissa Weber Mendonça championed the efforts to support NumPy users with visual impairments 
+and make the NumPy website and documentation more accessible: https://labs.quansight.org/blog/2021/07/numpy-accessibility-guidelines/.
```
Comment (jakevandervaate) [inline numpy-feature/numpy-accessibility.md]:
```suggestion
and make the NumPy website and documentation more accessible: [*Making Numpy Accessible: Guidelines and Tools*](https://labs.quansight.org/blog/2021/07/numpy-accessibility-guidelines/)
```

**Review by jakevandervaate — COMMENTED** [review]
Here are a few suggestions for small changes

File: numpy-feature/numpy-accessibility.md (hunk 1 above)
Comment (InessaPawson) [inline numpy-feature/numpy-accessibility.md]:
```suggestion
is often considered low-priority, scary, or someone else’s job. Read the blog post by Mars Lee, a front-end developer and designer at Quansight, 
```

**Pull request merged**